spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -204,6 +205,41 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
   }
 
   /**
+   * set trainable. If trainable is false, its parameters(weight/bias, if exists)
+   * are not changed in training process
+   * @param trainable whether this layer is trainable
+   */
+  def setTrainable(trainable: Boolean): this.type = {
```

Comment:
after user build a graph, he/she change the layer/trainable or stopGradient. Is this right?